Makefile

```diff
@@ -120,10 +120,11 @@ e2e-substitutions: $(ENVSUBST) $(subst $(E2E_TEMPLATES_DIR),$(E2E_ENVSUBST_DIR),
 	$(ENVSUBST) < $(E2E_CONF_FILE) > $(E2E_CONF_FILE_ENVSUBST)
 
 .PHONY: e2e-tests
+e2e-tests: CONTAINER_RUNTIME?=docker ## Env variable can override this default
 e2e-tests: e2e-substitutions ## This target should be called from scripts/ci-e2e.sh
-	docker pull quay.io/metal3-io/cluster-api-provider-metal3
-	docker pull quay.io/metal3-io/baremetal-operator
-	docker pull quay.io/metal3-io/ip-address-manager
+	$(CONTAINER_RUNTIME) pull quay.io/metal3-io/cluster-api-provider-metal3
+	$(CONTAINER_RUNTIME) pull quay.io/metal3-io/baremetal-operator
+	$(CONTAINER_RUNTIME) pull quay.io/metal3-io/ip-address-manager
```

Comment:
```suggestion
      @for image in $(SOME_LIST); do $(CONTAINER_RUNTIME) pull; done
```

Reply:
not necessarily important, so I leave it to you do decide which one to choose :)